### PR TITLE
Fixed an error with the ssh configuration

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

File: InstallCob4.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ function BasicInstallation {
   sudo apt-get install openssh-server -y --force-yes
   echo -e "\n${green}INFO:Let the server send a alive interval to clients to not get a broken pipe${NC}\n"
   echo "ClientAliveInterval 60" | sudo tee -a /etc/ssh/sshd_config
-  sudo sed -i 's/PermitRootLogin/PermitRootLogin yes/g' /etc/ssh/sshd_config
+  sudo sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/g' /etc/ssh/sshd_config
```
Comment (ipa-fxm) [inline InstallCob4.sh]: is this from a file name? or a string?
should the `SPACE` be made explicit (ASCII)?

Comment (ipa-nhg) [inline InstallCob4.sh]: the space is needed.
 `sed` find and replace strings with spaces

**Comment by ipa-nhg** [comment]
Test on cob4-2-h1  :white_check_mark:

**Comment by ipa-fmw** [comment]
CR: OK

**Pull request merged**